src/GitVersionCore.Tests/GitVersionCore.Tests.csproj

```diff
@@ -13,7 +13,8 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
-    <NuGetPackageImportStamp>4aaf7de2</NuGetPackageImportStamp>
+    <NuGetPackageImportStamp>
```

Comment:
Could you please drop the whole `<NuGetPackageImportStamp />` element? This is something that we shouldn't need in the committed .csproj files.